src/openzeppelin/account/IAccount.cairo

```diff
@@ -19,7 +19,7 @@ namespace IAccount {
     ) {
     }
 
-    func __validate_declare__(class_hash: felt) {
+    func __validate_declare__(cls_hash: felt) {
```

Comment:
Thanks for creating this PR, @unparalleled-js! Would you mind commenting here a link to the issue posted in the StarkWare repo and a quick note that it's a tmp fix?